libvuln/driver/updaterset.go

```diff
@@ -0,0 +1,87 @@
+package driver
+
+import (
+	"fmt"
+	"regexp"
+)
+
+// ErrExists is an error returned if the updater
+// already exists in the set.
+type ErrExists struct {
+	Updater []string
+}
+
+func (e ErrExists) Error() string {
+	return fmt.Sprintf("%v", e.Updater)
```

Comment:
This error string should have something more than a list literal.

Reply:
```
fmt.Sprintf("the following updaters already exist in the set: %+v", e.Updater)
```
?